util.py

```diff
@@ -298,9 +298,9 @@ def repair_robot(self, robot, state):
             'ground': Tile}
 
 
-def select_tile(direction, path, type, properties):
+def create_tile_subclass(direction, path, type, properties):
     """
-    Select tile subclass according to its type and create coressponding subclass.
+    Create tile subclass according to its type and create coressponding subclass.
```

Comment:
Bude to chtít přeformulovat ten docstring, teď je tam ve větě dvakrát `create`...

Reply:
Nějaký návrh?  :) ja bych to smazala za tim type ale nevim zda to odpovida

Reply:
Jo to by mohlo být, to dává smysl :)